openpower/package/petitboot/shell_profile

```diff
@@ -1,2 +1,10 @@
 export ENV="/home/petituser/.shrc"
-/usr/libexec/petitboot/pb-console
+TTY=${TTY:=`tty`}
+FBTERM=${FBTERM:=`which fbterm`}
+
+if [ -z "${TTY##/dev/tty*}" -a -x "$FBTERM" -a -w "/dev/fb0" ]
```

Comment:
You might want to be more specific here about the tty device: ${TTY##/dev/tty*} otherwise a serial port ( for example /dev/ttyS*) would fall into this rule, so maybe change it to ${TTY##/dev/tty[0-9]*}

Reply:
good catch.. I have also seen other things I need to fix, will respin this for another test soon, and will force-push here accordingly